components/ai_chat/core/browser/engine/conversation_api_client.h

```diff
@@ -80,6 +80,7 @@ class ConversationAPIClient {
     GetFocusTabsForTopic,
     PageScreenshot,
     ToolUse,
+    UserMemory,
```

Comment:
Out of scope for this PR, but I would like to bring attention that this should be an `enum class`, and the naming convention for these keys is not following the style guide suggestion: https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
https://github.com/brave/brave-browser/issues/47978